br.com.eduk.yml

```diff
@@ -0,0 +1,60 @@
+app-id: br.com.eduk
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: br.com.eduk
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/br.com.eduk/flatpak-node/cache
+        npm_config_cache: /run/build/br.com.eduk/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+        PUPPETEER_SKIP_DOWNLOAD: "true"
+    build-commands:
+      # Install npm dependencies
+      - npm install --offline
+      # Build the app; in this example the `dist` script
+      # in package.json runs electron-builder
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked /app/main
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ../run.sh
+      - install -Dm444 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm444 ./flatpak/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm444 ./flatpak/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    subdir: main
+    sources:
+      # Getting project
+      - type: git
+        url: https://github.com/greghorygusmao/br.com.eduk.app.git
```

Comment:
The link is 404... https://github.com/greghorygusmao/br.com.eduk.app

Reply:
Probably mean the repository is private.

Reply:
Fix